src/styles/zIndex.js

```diff
@@ -2,11 +2,12 @@
 // like global values in the browser.
 const zIndex = {
   mobileStepper: 1000,
-  appBar: 1100,
-  drawer: 1200,
-  modal: 1300,
-  snackbar: 1400,
-  tooltip: 1500,
+  speedDial: 1100,
```

Comment:
This is replicated in the README.md. It's a breaking change and a dangerous one as can have large impact without people noticing.